diff --git a/node_modules/@37signals/lexxy/dist/lexxy.esm.js b/node_modules/@37signals/lexxy/dist/lexxy.esm.js
index 91cc7b1..cad4266 100644
--- a/node_modules/@37signals/lexxy/dist/lexxy.esm.js
+++ b/node_modules/@37signals/lexxy/dist/lexxy.esm.js
@@ -564,7 +564,7 @@ class ActionTextAttachmentNode extends DecoratorNode {
     }
   }
 
-  constructor({ sgid, src, previewable, altText, caption, contentType, fileName, fileSize, width, height }, key) {
+  constructor({ sgid, src, previewable, altText, caption, contentType, fileName, fileSize, width, height } = {}, key) {
     super(key);
 
     this.sgid = sgid;
@@ -760,8 +760,8 @@ class ActionTextAttachmentUploadNode extends ActionTextAttachmentNode {
     return new ActionTextAttachmentUploadNode({ ...serializedNode })
   }
 
-  constructor({ file, uploadUrl, blobUrlTemplate, editor, progress }, key) {
-    super({ contentType: file.type }, key);
+  constructor({ file, uploadUrl, blobUrlTemplate, editor, progress } = {}, key) {
+    super({ contentType: file?.type }, key);
     this.file = file;
     this.uploadUrl = uploadUrl;
     this.blobUrlTemplate = blobUrlTemplate;
@@ -1944,7 +1944,7 @@ class CustomActionTextAttachmentNode extends DecoratorNode {
     }
   }
 
-  constructor({ sgid, contentType, innerHtml }, key) {
+  constructor({ sgid, contentType, innerHtml } = {}, key) {
     super(key);
 
     this.sgid = sgid;
